Makefile

```diff
@@ -21,16 +21,18 @@ merge: sol-merger-check
 	mkdir /tmp/sol-merger && mkdir ./contracts/sol-merger && \
 	sol-merger --export-plugin SPDXLicenseRemovePlugin ./contracts/FeralfileArtworkV2.sol /tmp/sol-merger && \
     sol-merger --export-plugin SPDXLicenseRemovePlugin ./contracts/FeralfileArtworkV3.sol /tmp/sol-merger && \
-    sol-merger --export-plugin SPDXLicenseRemovePlugin ./contracts/FeralfileArtworkV4_1.sol /tmp/sol-merger && \
+    sol-merger --export-plugin SPDXLicenseRemovePlugin ./contracts/FeralfileArtworkV4_2.sol /tmp/sol-merger && \
```

Comment:
Should keep the `V4_1` since it will be the current using contract. The `4_2` would just be one-off.